go.mod

```diff
@@ -4,11 +4,15 @@ go 1.12
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/elastic/beats v7.4.2+incompatible
```

Comment:
@urso @exekias I started to use `common.MapStr` in this PR which had not such a nice side effect dependency wise. I haven't check yet in detail but I guess it is because many things are in the common package so it adds all these :-( 

Should we extract common.MapStr in its own package?

Reply:
I'm actually thinking the same. Move often used functions/types into separate packages and use aliases to not break too much code. But MapStr is not one of my favorite types :), you really need it?

Reply:
Really need it is a bendable term ;-) The `GetValue` and `PutValue` that traverse the tree for dotted keys just make my code much simpler, otherwise I would have to reimplement that.